add-ons/fabric8/README.md

```diff
@@ -33,7 +33,7 @@ We currently recommend you use at least 5Gb of RAM in your minishift VM and toda
 
 So start minishift using something like this:
 
-    $ minishift start --memory=5000 --cpus=5 --disk-size=50g
+    $ minishift start --memory=5120 --cpus=5 --disk-size=50g
```

Comment:
:+1:  Sure will wait for https://github.com/minishift/minishift/issues/1068 to get resolved then we can add 5GB which will look more intuitive.